schema.graphqls

```diff
@@ -41,8 +41,26 @@ enum TrophyCategory {
     SKILL
 }
 
+"Interface for types that track some form of progression."
+interface ProgressionLike {
+    "The amount obtained."
+    obtained: Int!
+
+    "The amount that can be obtained."
+    obtainable: Int!
+}
+
+"A simple type tracking the progression of some data."
+type ProgressionData implements ProgressionLike {
```

Comment:
What’s the point of the type `ProgressionData` since it’s never used? 

Wouldn’t it make sense to remove? (Also due to being a duplicate of `ProgressionLike`)